src/content/docs/es/getting-started.mdx

```diff
@@ -99,8 +99,6 @@ Nuestra [guía de instalación](/es/install/auto/) tiene instrucciones completas
 
 [Historial de cambios de Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
 
-
-
```

Comment:
I’ll revert this change so we don’t accidentally mark as up-to-date based on the whitespace.